CHANGELOG.next.md

```diff
@@ -33,6 +33,7 @@ Thanks, you're awesome :-) -->
 #### Improvements
 
 * Go code generator now supports the `flattened` data type. #1302
+* Adjustments to use terminology that doesn't have negative connotation. #878
```

Comment:
Wouldn't we want the # to be the one of this PR (1315) ?

Reply:
Yes, let's use the PR number. Using the PR number lets someone trace the change back to where the implementation occurred.